src/builders/createBuilder.ts

```diff
@@ -0,0 +1,66 @@
+import type { BuilderApi, BuilderHandler, BuilderOptions, BuilderRegistration, IPipelineStage, Pipeline, PipelineStage } from '../@types'
```

Comment:
I'd personally prefer just to name it `types`. But I am also fine with this if you'd prefer.